spec/components/file_store/s3_store_spec.rb

```diff
@@ -420,6 +420,18 @@ def assert_path(path, expected)
 
       expect(store.signed_url_for_path("special/optimized/file.png")).not_to eq(upload.url)
     end
+
+    it "does not prefix the s3_bucket_folder_path onto temp/ keys" do
```

Comment:
```suggestion
    it "does not prefix the s3_bucket_folder_path onto #{FileStore::BaseStore::TEMPORARY_UPLOAD_PREFIX} keys" do
```

Reply:
Hmm this is a bit weird, do we usually interpolate strings into spec descriptions???

Reply:
Hmm maybe just say `temporary upload prefix` instead of duplicating the value of the constant in multiple places.

Reply:
Yep cool that makes more sense, I think the spec descriptions should be as static as possible :+1: